packages/core/src/components/FacetSearch/FacetSearch.tsx

```diff
@@ -0,0 +1,96 @@
+import { useState } from "react";
+import {
+  HvAccordion,
+  HvCheckBoxGroup,
+  HvCheckBox,
+  HvButton,
+  HvTypography,
+  HvInput,
+} from "@hitachivantara/uikit-react-core";
+import { css } from "@emotion/css";
+import { theme } from "@hitachivantara/uikit-styles";
+
+export const FacetSearch = () => {
```

Comment:
This is a bit confusing.
We're putting the sample in the same place we put components (`components/<CompName>/<CompName>.tsx`) and even exporting it in `index.ts`

Looking at how we do things elsewhere, this should probably be under a `stories` folder (ie. `stories/FacetSearch.tsx`)